### DISCUSSION TITLE
Valine and Waline configuration issue

### DISCUSSION THREAD

**Discussion by zhaoyiluo** [HEIGE-PCloud/DoIt#138] [Q&A] [answered]
### Describe the bug

I tested the Valine and Waline plugins separately to see if the visit statistics worked well. On my side, it only showed the eye symbol and didn't have the number, not even with 0. Everything goes well when I tested on Twikoo.

### Expected behavior

Show the visit number.

### Screenshots

![valine](https://user-images.githubusercontent.com/54724820/125171642-d7d9c400-e1e7-11eb-87ea-48c1a7a76a3e.png)

### Build Environment

- OS: Windows 10, iPadOS
- Theme version/commit: 0af16e1
- Hugo version: v0.84.4-020E4ACE+extended

### Preview Environment

https://www.imzluo.com/deepspeed/

### Additional Information

```bash
[params.page.comment.valine]
  enable = true
  appId = "aM7gLUeWp8it1Efrcp3rm8TB-MdYXbMMI"
  appKey = "Xn1xlYj3Fm5gvsYG3WyQpaix"
  placeholder = ""
  avatar = "mp"
  meta= ""
  pageSize = 10
  lang = "en"
  visitor = true
  recordIP = true
  highlight = true
  enableQQ = false
  serverURLs = ""
```

### Other

For the comment plugins, can we only enable the visit statistics and disable the comment part at the end of each post?

**Comment by HEIGE-PCloud** [comment, marked as answer]
Hi

You cannot just copy the `appId` and `appKey` from exampleSite. I have set up secure domains for these keys so Valine definitely will not work on your website with this configuration. Follow the instructions on [Valine Docs](https://valine.js.org/en/index.html) to setup your own App.

The visit statistics are recorded and provided by the comment plugin. So if you disable the comment, the statistics is disabled as well. The only thing you can do is to use some CSS style to hide the comment. You can edit `your_site/assets/css/_override.scss` and build your site with extended Hugo to achieve that. This will provide no performance improvement and I don't plan to add an option for it.

I know that there are some service providers offering only page view count statistics. I can probably add support for some of them if this is a wide need from community.
